addons/AsepriteWizard/animation_player/animation_creator.gd

```diff
@@ -150,6 +170,13 @@ func _add_animation_frames(target_node: Node, player: AnimationPlayer, anim_name
 	return result_code.SUCCESS
 
 
+const _INVALID_TOKENS := ["/", ":", ",", "["]
```

Comment:
This looks like dead code. Can you remove it?

Reply:
Ah, I forgot to actually use the validate function. I've added it to the main importer.